src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+# Spin a Wheel
+
+In the previous tutorial we used some simple functions to turn a light
+on and off.  Lets use some more advanced code to now control a motor.
+In this program we'll make use of a
+[VEX MC-29 motor controller](https://www.vexrobotics.com/276-2193.html).
+These motor controllers use a
+[Pulse Width Modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation)
+signal to determine direction and magnitude of the motor rotation they
+control.
+
+For this program, we're going to use two new modules: `pwmio` and
+`adafruit_motor`.
+
+## The pwmio Module
+
+PWM is a simple way to send an analog signal over a digital pin. It is
+commonly used for controlling motors, servos, and other actuators. The
+`pwmio` module provides the `PWMOut` class to efficiently generate PWM
+signals on a pin using our microcontroller's built-in timers. There are
+two important values for us to control with a `PWMOut` object: frequency
+and duty cycle.
+
+Frequency is the number of times each second that the signal changes from
+low to high and back to low. This is measured in Hertz (Hz).
+
+Duty Cycle is the fraction of the total pulse time where the signal is
+held high. 0% duty cycle is a signal that is always low. 100% duty cycle
+is a signal that is always high. 50% duty cyclce is a signal that spends
+an equal amount of time high and low.
+
+While PWM is an important concept to understand in robotics, we don't need
+to use it directly for controlling motors. We introduce this module here
+because it is used by the `adafruit_motor` module.
+
+API documentation for the `pwmio` module is available
+[here](https://docs.circuitpython.org/en/latest/shared-bindings/pwmio/index.html).
+
+## The adafruit_motor Module
+
+The `adafruit_motor` module provides tools for controlling a variety of
+types of motors. We will use the `adafruit_motor.servo` module specifically.
+
+Why would we use a "servo" module for controlling our motors? In our case,
```

Comment:
Generally this documentation should avoid direct mention of the BEST rkit since that is in a perpetual state of flux year on year.  Consider embedding a picture here of the specific kind of cable you want people to draw the relation to, and perhaps note that this is a standard in the R/C world.